Nie-ontologies/Generic-ontologies/organization.ttl

```diff
@@ -89,7 +89,7 @@ org:Politician
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/organization#>;
 	a rdfs:Class;
 	#rdfs:isDefinedBy <http://e-editiones.ch/ontology/organization#>;
-	rdfs:label "politician role"@en, "Politiker-Rolle"@de;
+	rdfs:label "politician role"@en, "Rolle eines Politikers"@de;
```

Comment:
politician role -> politician role of person; Rolle eines Politikers -> Politikerrolle einer Person

Reply:
Ok.

Reply:
Commit a34d6e